xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-wysiwyg/xwiki-platform-realtime-wysiwyg-test/xwiki-platform-realtime-wysiwyg-test-docker/src/test/it/org/xwiki/realtime/wysiwyg/test/ui/RealtimeWYSIWYGEditorIT.java

```diff
@@ -641,4 +643,67 @@ void imageWithCaption(TestReference testReference, TestUtils setup) throws Excep
         assertEquals("before\n\n[[Smallest **Tree**>>image:image.gif]]\n\n ",
             WikiEditPage.gotoPage(testReference).getContent());
     }
+    
+    @Test
+    @Order(8)
+    void reloadEditorsMergeConflictManualSave(TestReference testReference, TestUtils setup)
+    {
+        //
+        // First Tab
+        //
+
+        // Start fresh.
+        setup.deletePage(testReference);
+
+        RealtimeWYSIWYGEditPage firstEditPage = RealtimeWYSIWYGEditPage.gotoPage(testReference);
+        RealtimeCKEditor firstEditor = firstEditPage.getContenEditor();
+        RealtimeRichTextAreaElement firstTextArea = firstEditor.getRichTextArea();
+        firstTextArea.sendKeys("First");
+        firstEditPage.clickSaveAndContinue();
+
+        //
+        // Second Tab
+        //
+
+        String secondTabHandle = setup.getDriver().switchTo().newWindow(WindowType.TAB).getWindowHandle();
+
+        RealtimeWYSIWYGEditPage secondEditPage = RealtimeWYSIWYGEditPage.gotoPage(testReference);
+        RealtimeCKEditor secondEditor = secondEditPage.getContenEditor();
+        RealtimeRichTextAreaElement secondTextArea = secondEditor.getRichTextArea();
+
+        secondTextArea.waitUntilContentContains("First");
+
+        //
+        // Third Tab
+        //
+
+        String thirdTabHandle = setup.getDriver().switchTo().newWindow(WindowType.TAB).getWindowHandle();
+
+        RealtimeWYSIWYGEditPage thirdEditPage = RealtimeWYSIWYGEditPage.gotoPage(testReference);
+        RealtimeCKEditor thirdEditor = thirdEditPage.getContenEditor();
+        RealtimeRichTextAreaElement thirdTextArea = thirdEditor.getRichTextArea();
+        thirdTextArea.waitUntilContentContains("First");
+
+        thirdEditPage.leaveRealtimeEditing();
+
+        thirdTextArea.sendKeys(Keys.END, " Second");
+        thirdEditPage.clickSaveAndContinue();
+
+        // First tab
+
+        setup.getDriver().switchTo().window(firstTabHandle);
+        firstTextArea.sendKeys(Keys.END, " First");
+        firstTextArea.waitUntilContentContains("First First");
```

Comment:
This is intended to make sure that the content is what we expect it to be. It could be something else if the third tab failed to exit realtime for instance. But I think other scenarios where the content is not the expected one are imaginable.
Thanks